schedule/functional/extra_tests_textmode.yaml

```diff
@@ -80,4 +128,5 @@ schedule:
     - console/aaa_base
     - console/osinfo_db
     - console/zypper_ref
```

Comment:
zypper_ref here, would be redundant now

Reply:
zypper_ref is executed multiple times in the original ´extra_tests_in_textmode`